src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./TodoItem.scss";
+import { useState, useContext } from "react";
+import { apiBaseUrl, requestDefaultHeaders } from "../../../config";
+import { StatusLengthContext } from "../../../StatusLengthContext";
+
+const TodoItem = ({ text, createdAt, taskId, tasks, setTasks }) => {
+  const [error, setError] = useState(null);
+  const { allLength, setAllLength } = useContext(StatusLengthContext);
+
+  const deleteTodo = () => {
```

Comment:
Before trying deletion. Clean up the error state.

Reply:
Done.